### PR TITLE
[daint] [dom] Python-bare 3.6.2 for craype/17.08

### PR DESCRIPTION
**Pull request by vkarak**

### CODE REVIEW

File: jenkins-builds/6.0.UP04-17.08-mc [hunk 1]
```diff
@@ -8,3 +8,4 @@
  IPM-2.0.6-CrayIntel-17.08.eb
  IPM-2.0.6-CrayPGI-17.08.eb
  LAMMPS-11Aug2017-CrayGNU-17.08.eb                  --set-default-module
+ Python-bare/.3.6.2
```
Comment (lucamar) [inline jenkins-builds/6.0.UP04-17.08-mc]: You need to write the EasyBuild recipe name with the extension `.eb` and use the flag `--hidden` to hide it, e.g.:
`Python-bare-3.6.2.eb --hidden`

File: jenkins-builds/6.0.UP04-17.08-gpu [hunk 2]
```diff
@@ -10,6 +10,7 @@
  IPM-2.0.6-CrayIntel-17.08.eb
  IPM-2.0.6-CrayPGI-17.08.eb
  LAMMPS-11Aug2017-CrayGNU-17.08-cuda-8.0.eb         --set-default-module
+ Python-bare/.3.6.2
```
Comment (lucamar) [inline jenkins-builds/6.0.UP04-17.08-gpu]: You need to write the EasyBuild recipe name with the extension `.eb` and use the flag `--hidden` to hide it, e.g.:
`Python-bare-3.6.2.eb --hidden`

File: jenkins-builds/6.0.UP04-17.08-mc (hunk 1 above)
Comment (vkarak) [reply jenkins-builds/6.0.UP04-17.08-mc]: Should I also set is as default?

Comment (lucamar) [reply jenkins-builds/6.0.UP04-17.08-mc]: No

**Comment by vkarak** [comment]
retest this please

**Comment by vkarak** [comment]
retest this please

**Comment by lucamar** [comment]
@vkarak You may want to test the build on `dom` as well: in this case, please add `dom` to the Pull Request title.

**Comment by vkarak** [comment]
It should work on Dom, since I've tested it there. As soon as this is merged, I will do another PR for Reframe.

**Comment by vkarak** [comment]
please retest this

**Pull request merged**